duo_unix_support/duo_unix_support.sh

```diff
@@ -112,6 +112,14 @@ else
     echo "Could not find version of Duo Unix (login_duo was not found)" > configuration.txt
 fi
 
+if [ "$OS" = "solaris" ]; then
+    if type ggrep >/dev/null; then
```

Comment:
Consider adding `2>&1` to this line. Some shells write type's error messages to stdout and some to stderr.